<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimeEpisodeWidgetItem</class>
 <widget class="QWidget" name="AnimeEpisodeWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
  border-left: 5px solid #ff0000;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_See">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_See
{
  border: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Resource.qrc">
          <normaloff>:/File/img/check_it.png</normaloff>
          <normalon>:/File/img/check_ok.png</normalon>:/File/img/check_it.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Tag1">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Tag2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Tag3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Name">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string notr="true">显示名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
